packages/dnb-eufemia/src/extensions/forms/Field/Date/__tests__/Date.test.tsx

```diff
@@ -371,6 +371,54 @@ describe('Field.Date', () => {
     ).toHaveAttribute('aria-disabled', 'true')
   })
 
+  it('should support min and max dates as date objects', async () => {
```

Comment:
```suggestion
  it('should validate min and max dates as days when providing date objects', async () => {
```
Is this better? Not sure 🤔